keyboards/themadnoodle/noodlepad_micro/config.h

```diff
@@ -3,4 +3,4 @@
 
 #pragma once
 
-#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL + 5
+#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL + 5
```

Comment:
Keep the newline at EOF here please.

Reply:
I added a newline to the end of the file is that what you mean?

Also, can we get this resolved and merged as soon as we can this has been marked as stale and will be closed even though I've been making the changes as soon as they are requested of me.